### ISSUE TITLE
Makes "dmarc_rua" optional

### ISSUE THREAD

**Issue by wpsantana** [trussworks/terraform-aws-ses-domain#110]
Hello, I would like to thank you so much for building a great module.

So I would like to ask if is possible to make `dmarc_rua` optional because we can have use cases when we need to create more than one AWS SES (i.e one in each AZ) for the same domain.

Suggestion:

```
resource "aws_route53_record" "txt_dmarc" {
  count = var.enable_dmarc ? 1 : 0

  zone_id = var.route53_zone_id
  name    = "_dmarc.${var.domain_name}"
  type    = "TXT"
  ttl     = "600"
  records = ["v=DMARC1; p=${var.dmarc_p}; rua=mailto:${var.dmarc_rua};"]
```

A bunch of thanks.

**Comment by rpdelaney** [comment]
@wpsantana Humblest apologies for the late turn-around. I see no problem with this. We will put it on the road map, but in the mean time a PR would be welcome :)

**Comment by esacteksab** [comment]
PR created here https://github.com/trussworks/terraform-aws-ses-domain/pull/130

**Comment by rpdelaney** [comment]
Shipped in `v3.2.3`!
